### ISSUE TITLE
Unable to run the migration file - Connection.php `could not find driver`

### ISSUE THREAD

**Issue by mamdooh26** [susanBuck/dwa15-spring2018#57]
The connection with DB is working correctly when I added the debug method.
![image](https://user-images.githubusercontent.com/35989352/38666788-778fb8b6-3e48-11e8-8007-e01c64b34dc7.png)

But when I tried to run the migration files, I faced the below error:

![image](https://user-images.githubusercontent.com/35989352/38666754-64c63c6e-3e48-11e8-8e4a-f7639ffe3fc7.png)

Your help is highly appreciated.

**Comment by susanBuck** [comment]
From your screenshot, it looks like you're using MAMP on Windows? 

If this is the case, check out this post from last semester: [Local Database Config - Error: FAILED: could not find driver](https://github.com/susanBuck/dwa15-fall2017/issues/129#issuecomment-342359508).

Specifically, the solution was in [this reply](https://github.com/susanBuck/dwa15-fall2017/issues/129#issuecomment-342359508) I made to that post.

If that's not the case, let us know and we can look into other possible solutions.

**Comment by archerdave** [comment]
Ah, I see the difference.  Got it!  Thanks!

**Comment by mamdooh26** [comment]
I tried it but unfortunately it didn't fix the issue.

**Comment by susanBuck** [comment]
@mamdooh26 Can you confirm if you're using MAMP on Windows? If not, what setup are you using?

**Comment by mamdooh26** [comment]
Yes I'm using MAMP on Windows

**Comment by susanBuck** [comment]
MAMP on Windows was not one of the platforms supported in this course; things should go smoother if you stick with the suggested software (XAMPP for Windows).

That being said, I can still work with you to try and sort this out of you're really wanting to use MAMP on Windows, but I'm going to need more details, for example:

+ What steps have you taken to try and resolve this issue?
+ What posts have you found online related to this problem?
+ When following the instructions in the post I shared above, did you run into any issues with the process? Was there any part of the process that I described that was different from what you saw on your end?

In short, I need more info, because it's hard to troubleshoot something I don't directly have access to. 

Also, if possible, you may want to sign up for office hours so we can debug this in real-time. If my listed office hours don't work for you, reach out and we can set up an alternative time.

**Comment by susanBuck** [comment]
Closing this post due to inactivity, but feel free to continue the discussion or post a new thread if any problems are still pending.
